modules/ballerina-core/src/main/java/org/ballerinalang/model/values/BJSON.java

```diff
@@ -110,7 +110,7 @@ public BJSON(String jsonString, String schema) {
                 this.schema = OBJECT_MAPPER.readTree(schema);
             }
         } catch (Throwable t) {
-            handleJsonException("failed to create json: ", t);
+            handleJsonException(null, t);
```

Comment:
Here passing a message causes to log duplicate information in an error. e.g: `failed to parse json: failed to create json: ...` . hence only propagating the actual error message only. 

Fixed the null passing, by overloading the method.